<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProxyPage</class>
 <widget class="QWidget" name="ProxyPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>617</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <widget class="QWidget" name="tabWidgetPage1">
      <attribute name="title">
       <string notr="true"/>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="proxyPlainTextWarningLabel_2">
         <property name="text">
          <string>This only applies to the launcher. Minecraft does not accept proxy settings.</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="proxyTypeBox">
         <property name="title">
          <string>Type</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QRadioButton" name="proxyDefaultBtn">
            <property name="toolTip">
             <string>Uses your system's default proxy settings.</string>
            </property>
            <property name="text">
             <string>&amp;Default</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">proxyGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="proxyNoneBtn">
            <property name="text">
             <string>&amp;None</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">proxyGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="proxySOCKS5Btn">
            <property name="text">
             <string>SOC&amp;KS5</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">proxyGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="proxyHTTPBtn">
            <property name="text">
             <string>H&amp;TTP</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">proxyGroup</string>
            </attribute>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="proxyAddrBox">
         <property name="title">
          <string>Address and Port</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLineEdit" name="proxyAddrEdit">
            <property name="placeholderText">
             <string notr="true">127.0.0.1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="proxyPortEdit">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::PlusMinus</enum>
            </property>
            <property name="maximum">
             <number>65535</number>
            </property>
            <property name="value">
             <number>8080</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="proxyAuthBox">
         <property name="title">
          <string>Authentication</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="1">
           <widget class="QLineEdit" name="proxyUserEdit"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="proxyUsernameLabel">
            <property name="text">
             <string>Username:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="proxyPasswordLabel">
            <property name="text">
             <string>Password:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="proxyPassEdit">
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QLabel" name="proxyPlainTextWarningLabel">
            <property name="text">
             <string>Note: Proxy username and password are stored in plain text inside the launcher's configuration file!</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="proxyGroup"/>
 </buttongroups>
</ui>
